Added doc for switching elasticsearch to infinity (#3370)
infiniflow/ragflow: docs/guides/develop/build_docker_image.mdx

```

  </TabItem>
  <TabItem value="linux/arm64">

## 🔧 Build a Docker image for linux arm64

> We have difficulty on building multi-arch image with CI. So we will not publish arm64 image officially in the near future.  

Reviewer: ```suggestion
We cannot build multi-architecture images with CI and do not plan to publish arm64 images in the near future.  
```

Author: done

> However you can build it by yourself on a `linux/arm64` host:

Reviewer: ```suggestion
However, you can build an image yourself on a linux/arm64 host machine:
```
